packages/models-library/src/models_library/user_preferences.py

```diff
@@ -93,6 +93,13 @@ class FrontendUserPreference(_BaseUserPreferenceModel):
     def to_db(self) -> dict:
         return self.dict(exclude={"preference_identifier", "preference_type"})
 
+    @classmethod
+    def get_preference_identifier(cls) -> PreferenceIdentifier:
+        preference_identifier: PreferenceIdentifier = cls.__fields__[
+            "preference_identifier"
+        ].default
```

Comment:
1. This returns the default value but you call it `get_preference_identifier`
1. the **default** of the `preference_identifier` field in this class is going to be `None`! Example
```
>>> from pydantic import *
>>> class A(BaseModel):
...    x: int = Field(...)
... 
>>> A.__fields__["x"].default
>>> A.__fields__["x"].default is None
True
```